spec/unit/lib/shared/utils.spec.js

```diff
@@ -0,0 +1,293 @@
+/*
+
+ ----------------------------------------------------------------------------
+ | ripple-fhir-service: Ripple FHIR Interface                               |
+ |                                                                          |
+ | Copyright (c) 2017-19 Ripple Foundation Community Interest Company       |
+ | All rights reserved.                                                     |
+ |                                                                          |
+ | http://rippleosi.org                                                     |
+ | Email: code.custodian@rippleosi.org                                      |
+ |                                                                          |
+ | Author: Richard Brown                                                    |
+ |                                                                          |
+ | Licensed under the Apache License, Version 2.0 (the 'License');          |
+ | you may not use this file except in compliance with the License.         |
+ | You may obtain a copy of the License at                                  |
+ |                                                                          |
+ |     http://www.apache.org/licenses/LICENSE-2.0                           |
+ |                                                                          |
+ | Unless required by applicable law or agreed to in writing, software      |
+ | distributed under the License is distributed on an 'AS IS' BASIS,        |
+ | WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. |
+ | See the License for the specific language governing permissions and      |
+ |  limitations under the License.                                          |
+ ----------------------------------------------------------------------------
+
+  18 March 2019
+
+*/
+
+'use strict';
+
+const { 
+  getPractitionerRef, 
+  parseTelecom, 
+  parseAddress,
+  getOrganisationRef,
+  parseName
+} = require('@lib/shared/utils');
+
+describe('ripple-cdr-lib/lib/shared/utils', () => {
```

Comment:
@richardcbrown is "ripple-cdr-lib" the correct name here? (and in other tests)? I'd rather recommend you to use `lib/shared/utils` here. Basically, with current folders structure you should have 2 root folders in `spec/unit` folder -> `apis` and `lib`